RallyHereIntegration/Source/RallyHereIntegration/Public/RH_IntegrationSettings.h

```diff
@@ -273,6 +273,20 @@ class RALLYHEREINTEGRATION_API URH_IntegrationSettings : public UDeveloperSettin
 
 	static bool ShouldUseLocalPlayerSandboxing();
 
+	////////////////////////////////////////////////////////////////////////////////////////////////////////
+	// Automatic File Uploads
+	////////////////////////////////////////////////////////////////////////////////////////////////////////
+	/** @brief Whether to automatically upload files to the RallyHere API from Dedicated Servers */
+	UPROPERTY(EditAnywhere, Config, Category = "File Uploads")
+	bool bAutoUploadServerFiles;
+	/** @brief Whether to automatically upload files to the RallyHere API.  Requires bAutoUploadServerFiles to have an effect */
+	UPROPERTY(EditAnywhere, Config, Category = "File Uploads", meta=(EditCondition=bAutoUploadFiles))
```

Comment:
The edit condition for these two values looks like they don't match the variable above

Reply:
Oops, missed them on the rename pass.  Surprised UHT didnt complain.